### PR TITLE
<amp-facebook-*>Add data-locale attribute

### LINKED ISSUE #9264: amp-facebook-*: support different languages
Currently we always load the `US_en` SDK. At minimum it should load the one based on document `lang` and also maybe allow overriding the locale at element level by introducing `data-locale` on all `amp-facebook-*` components.

--- Originally reported by @cvrlebg :
## What's the issue?

Hi, can you add new attribute for amp-facebook-like, something like data-locale where we can add language code like on FB documentation page: [https://developers.facebook.com/docs/plugins/like-button#language](https://developers.facebook.com/docs/plugins/like-button#language)?

## How do we reproduce the issue?

This will be very useful for non-english sites.

Thanks!

### PR DESCRIPTION
**Pull request by nainar**
This also adds examples, tests and explanations for the new attribute. 

Fixes #9264

### CODE REVIEW

**Comment by nainar** [comment]
Status: Not ready for review yet.

**Comment by nainar** [comment]
Ready for review - @aghassemi PTAL?

File: extensions/amp-facebook-comments/amp-facebook-comments.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ You can use the `amp-facebook-comments` component to embed the [Facebook comment
 
 The URL of the comments page. For example, `http://www.directlyrics.com/adele-25-complete-album-lyrics-news.html`.
 
+##### data-locale (optional)
+
+By default, the local is set to <code>en_US</code>; however, you can specify a locale as well.
```
Comment (aghassemi) [inline extensions/amp-facebook-comments/amp-facebook-comments.md]: Default is `user's system language` right?

File: extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii [hunk 2]
```diff
@@ -33,6 +33,7 @@ tags: {  # <amp-facebook-comments>
     name: "data-href"
     mandatory: true
   }
+  attrs: { name: "data-locale" }
```
Comment (aghassemi) [inline extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii]: no need, `data-*` attributes are always white-listed. We only list them in validation rules if they need to be required or have a regex check their value.

File: extensions/amp-facebook-comments/amp-facebook-comments.md (hunk 1 above)
Comment (aghassemi) [reply extensions/amp-facebook-comments/amp-facebook-comments.md]: ditto other components

File: extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii (hunk 2 above)
Comment (aghassemi) [reply extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii]: ditto other components

File: examples/facebook.amp.html [hunk 3]
```diff
@@ -25,6 +25,12 @@ <h2>Facebook</h2>
 
 <h1>More Posts</h1>
 
+<amp-facebook width=552 height=303
```
Comment (aghassemi) [inline examples/facebook.amp.html]: I don't see any changes to `getFacebookSdk()` in `facebook.js` , a bit confused on how this is working, missed a commit?

Comment (nainar) [reply examples/facebook.amp.html]: Added the missing file. Sorry about that!

File: extensions/amp-facebook-comments/amp-facebook-comments.md (hunk 1 above)
Comment (nainar) [reply extensions/amp-facebook-comments/amp-facebook-comments.md]: Done

File: extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii (hunk 2 above)
Comment (nainar) [reply extensions/amp-facebook-comments/validator-amp-facebook-comments.protoascii]: Done

**Pull request merged**